.travis.yml

```diff
@@ -0,0 +1,34 @@
+# while this doesn't require sudo we don't want to run within a Docker container
+sudo: true
+dist: trusty
```

Comment:
Don't think it impacts anything given that the job will run in docker but should we make this xenial ?